.github/workflows/main.yml

```diff
@@ -50,10 +50,6 @@ jobs:
         shell: bash -l {0}
         run: |
           conda activate test
-          coverage run run_tests.py
```

Comment:
Do we not need `run_tests.py` anymore then?

It felt awkward to have a separate file for running pytest, indeed.